hydromt_wflow/wflow.py

```diff
@@ -5036,43 +5033,6 @@ def write_config(
         # Call the component
         self.config.write(p)
 
-    @hydromt_step
-    def read_grid(self, **kwargs):
```

Comment:
You deleted the read but kept the write and set? I was not there when the decision was made but good to be consistent

Reply:
Should be back in (read_staticmaps)